tests/pod.py

```diff
@@ -97,19 +97,31 @@ def check_for_restarted_pods(namespace, specific_deployment_name=''):
     return restarted_pods
 
 
-def search_phrase_in_pod_log(pod_name, name_space, container_name, phrase, time_out=60, group=None):
-
-    match = None
+def search_phrase_in_pod_log(pod_name, name_space, container_name, phrase, timeout=60, group=None, retry=3):
+    sleep_interval = 0.3
     total_sleep = 0
+    _retry = retry
```

Comment:
why this? we can't just change retry?

Reply:
not really important though but confusing

Reply:
@y0sher this var is reset in line 125 if a successful call was made but no match found.

@AmitShaul I would consider renaming `_retry` to `retries_remaining` and `retry` to `retries`.